data_entry/how-to-enter-recency-data.md

```diff
@@ -0,0 +1,23 @@
+## How To Enter Recency Data Within The Hts Client Card
+
+### Accessing the HTS client card for an existing or new patient in the EMR
+
+1.	Search for the given patient or create a new patient
+
+2.	Open a recent visit or start a past visit for the newly created patient
+
+3.	View the patient’s dashboard and look for the HMIS ACP 018:HTS CLIENT CARD that shall be used to enter in the data.
+
+![HTS Client Card](../images/data-entry/hts-client-card/hts-client-card018.jpg)
+
+### Notes.
```

Comment:
@ODORA0 Remove the full stop at the end of the heading

Reply:
@ssmusoke Done